packages/sql/src/internal/statement.ts

```diff
@@ -130,6 +130,13 @@ export class StatementPrimitive<A> extends Effectable.Class<ReadonlyArray<A>, Er
     )
   }
 
+  get raw(): Effect.Effect<unknown, Error.SqlError> {
+    return this.withConnection(
+      "executeRaw",
+      (connection, sql, params) => connection.executeRaw(sql, params)
+    )
+  }
```

Comment:
```suggestion
  get raw(): Effect.Effect<unknown, Error.SqlError> {
    return this.withConnection(
      "executeRaw",
      (connection, sql, params) => connection.executeRaw(sql, params),
      true
    )
  }
```

Remove the transformations too.